app/views/devise/sessions/new.html.erb

```diff
@@ -47,6 +47,7 @@
           },
         },
       ) %>
+  <%= hidden_field_tag :platform_authenticator_available, '', id: 'platform_authenticator_available' %>
```

Comment:
would it help at all to make the default value like `'unknown'` or something, to differentiate from the true/false from the JS? ex if the JS doesn't run